diff --git a/Lifestream/Lifestream/Systems/Legacy/DataStore.cs b/Lifestream/Lifestream/Systems/Legacy/DataStore.cs
index da3eb1f..efab9a7 100644
--- a/Lifestream/Lifestream/Systems/Legacy/DataStore.cs
+++ b/Lifestream/Lifestream/Systems/Legacy/DataStore.cs
@@ -114,9 +114,9 @@ public class DataStore
 
     internal void BuildWorlds(uint dc)
     {
-        Worlds = [.. Svc.Data.GetExcelSheet<World>().Where(x => x.DataCenter.Value.RowId == dc && x.IsPublic()).Select(x => x.Name.ToString()).Order()];
+        Worlds = [.. Svc.Data.GetExcelSheet<World>().Where(x => x.DataCenter.Value.RowId == dc && Utils.IsTravelWorld(x)).Select(x => x.Name.ToString()).Order()];
         PluginLog.Debug($"Built worlds: {Worlds.Print()}");
-        DCWorlds = Svc.Data.GetExcelSheet<World>().Where(x => x.DataCenter.Value.RowId != dc && x.IsPublic() && (x.DataCenter.Value.Region == Player.Object.HomeWorld.Value.DataCenter.Value.Region || x.DataCenter.Value.Region == 4)).Select(x => x.Name.ToString()).ToArray();
+        DCWorlds = Svc.Data.GetExcelSheet<World>().Where(x => x.DataCenter.Value.RowId != dc && Utils.IsTravelWorld(x) && (x.DataCenter.Value.Region == Player.Object.HomeWorld.Value.DataCenter.Value.Region || x.DataCenter.Value.Region == 4)).Select(x => x.Name.ToString()).ToArray();
         PluginLog.Debug($"Built DCworlds: {DCWorlds.Print()}");
     }
 
diff --git a/Lifestream/Lifestream/Utils.cs b/Lifestream/Lifestream/Utils.cs
index fa69ce2..ad12b2b 100644
--- a/Lifestream/Lifestream/Utils.cs
+++ b/Lifestream/Lifestream/Utils.cs
@@ -83,6 +83,29 @@ internal static unsafe partial class Utils
         return input;
     }
 
+    public static bool IsTravelWorld(World world)
+    {
+        if(world.IsPublic())
+        {
+            return true;
+        }
+
+        if(world.DataCenter.Value.RowId == 151 &&
+           TravelWorldAliases.Keys.Contains(world.Name.ToString(), StringComparer.OrdinalIgnoreCase))
+        {
+            return true;
+        }
+
+        return false;
+    }
+
+    public static World[] GetTravelWorlds(uint dataCenter)
+    {
+        return Svc.Data.GetExcelSheet<World>()
+            .Where(x => x.DataCenter.RowId == dataCenter && IsTravelWorld(x))
+            .ToArray();
+    }
+
     public static Vector3 Scatter(this Vector3 point, float radius)
     {
         if(radius > 0)
@@ -802,7 +825,7 @@ internal static unsafe partial class Utils
             if(x.RowId == 0 || x.Name == "") continue;
             if(x.Name.GetText().StartsWith(s, StringComparison.OrdinalIgnoreCase))
             {
-                var worlds = ExcelWorldHelper.GetPublicWorlds(x.RowId);
+                var worlds = GetTravelWorlds(x.RowId);
                 if(worlds.Length > 0)
                 {
                     world = worlds[Random.Shared.Next(worlds.Length)].Name.ToString();
