diff --git a/Lifestream/OtterGui/Classes/FilterUtility.cs b/Lifestream/OtterGui/Classes/FilterUtility.cs
index bbb7572..7e3a2fd 100644
--- a/Lifestream/OtterGui/Classes/FilterUtility.cs
+++ b/Lifestream/OtterGui/Classes/FilterUtility.cs
@@ -20,7 +20,7 @@ public abstract class FilterUtility<T>
     public bool Draw(float width)
     {
         ImGui.SetNextItemWidth(width);
-        var change = ImGui.InputTextWithHint("##filterInput", "Filter...", ref _input, 64);
+        var change = ImGui.InputTextWithHint("##filterInput", "篩選...", ref _input, 64);
         ImGuiUtil.HoverTooltip(Tooltip);
         if (!change)
             return false;
diff --git a/Lifestream/OtterGui/Classes/StartTimeTracker.cs b/Lifestream/OtterGui/Classes/StartTimeTracker.cs
index a4e3f90..9a25088 100644
--- a/Lifestream/OtterGui/Classes/StartTimeTracker.cs
+++ b/Lifestream/OtterGui/Classes/StartTimeTracker.cs
@@ -83,7 +83,7 @@ public class StartTimeTracker
     public void Draw(string label)
     {
         using var id = ImRaii.PushId(label);
-        UserRegex.DrawRegexInput("##filter", ref _filter, "Filter...", null, ImGui.GetContentRegionAvail().X, 0xFF4040F0);
+        UserRegex.DrawRegexInput("##filter", ref _filter, "篩選...", null, ImGui.GetContentRegionAvail().X, 0xFF4040F0);
         using var table = ImRaii.Table("##startTimeTable", 5, ImGuiTableFlags.SizingFixedFit | ImGuiTableFlags.RowBg);
 
         ImGui.TableSetupColumn("Name",   ImGuiTableColumnFlags.None, 150 * ImGuiHelpers.GlobalScale);
diff --git a/Lifestream/OtterGui/Filesystem/Selector/FileSystemSelector.State.cs b/Lifestream/OtterGui/Filesystem/Selector/FileSystemSelector.State.cs
index 63c5665..b37b0b9 100644
--- a/Lifestream/OtterGui/Filesystem/Selector/FileSystemSelector.State.cs
+++ b/Lifestream/OtterGui/Filesystem/Selector/FileSystemSelector.State.cs
@@ -66,7 +66,7 @@ public partial class FileSystemSelector<T, TStateStorage> : IDisposable
         ImGui.SetNextItemWidth(width);
         var       tmp    = FilterValue;
         using var id     = ImRaii.PushId(0, clear);
-        var       change = ImGui.InputTextWithHint("##Filter", "Filter...", ref tmp, 128);
+        var       change = ImGui.InputTextWithHint("##Filter", "篩選...", ref tmp, 128);
         if (ImGui.IsItemClicked(ImGuiMouseButton.Right) && !ImGui.IsItemFocused())
         {
             try
diff --git a/Lifestream/OtterGui/Filesystem/Selector/FileSystemSelector.cs b/Lifestream/OtterGui/Filesystem/Selector/FileSystemSelector.cs
index 92398b5..72e2173 100644
--- a/Lifestream/OtterGui/Filesystem/Selector/FileSystemSelector.cs
+++ b/Lifestream/OtterGui/Filesystem/Selector/FileSystemSelector.cs
@@ -349,7 +349,7 @@ public partial class FileSystemSelector<T, TStateStorage> where T : class where
         ImGui.SetNextItemWidth(width);
         var tmp = FilterValue;
         using var id = ImRaii.PushId(0, clear);
-        var change = ImGui.InputTextWithHint("##Filter", "Filter...", ref tmp, 128);
+        var change = ImGui.InputTextWithHint("##Filter", "篩選...", ref tmp, 128);
         if (ImGui.IsItemClicked(ImGuiMouseButton.Right) && !ImGui.IsItemFocused())
         {
             try
diff --git a/Lifestream/OtterGui/ItemSelector.cs b/Lifestream/OtterGui/ItemSelector.cs
index d0b6bda..271a654 100644
--- a/Lifestream/OtterGui/ItemSelector.cs
+++ b/Lifestream/OtterGui/ItemSelector.cs
@@ -261,7 +261,7 @@ public class ItemSelector<T>
         var       newFilter = Filter;
         using var style     = ImRaii.PushStyle(ImGuiStyleVar.FrameRounding, 0);
         ImGui.SetNextItemWidth(width);
-        var enterPressed = ImUtf8.InputText("##"u8, ref newFilter, "Filter..."u8, ImGuiInputTextFlags.EnterReturnsTrue);
+        var enterPressed = ImUtf8.InputText("##"u8, ref newFilter, "篩選..."u8, ImGuiInputTextFlags.EnterReturnsTrue);
         if (newFilter != Filter)
         {
             Filter      = newFilter;
@@ -309,7 +309,7 @@ public class ItemSelector<T>
         if (ImGui.Button(FontAwesomeIcon.Plus.ToIconString(), Vector2.UnitX * width))
             ImGui.OpenPopup(newNamePopupAdd);
         using var font = ImRaii.PushFont(UiBuilder.DefaultFont);
-        ImGuiUtil.HoverTooltip("Add New");
+        ImGuiUtil.HoverTooltip("新增");
 
         if (!OpenNameField(newNamePopupAdd, out var newName))
             return;
diff --git a/Lifestream/OtterGui/Widgets/ClippedSelectableCombo.cs b/Lifestream/OtterGui/Widgets/ClippedSelectableCombo.cs
index 00ac256..4307738 100644
--- a/Lifestream/OtterGui/Widgets/ClippedSelectableCombo.cs
+++ b/Lifestream/OtterGui/Widgets/ClippedSelectableCombo.cs
@@ -72,7 +72,7 @@ public class ClippedSelectableCombo<T>
 
         ImGui.SetNextItemWidth(-1);
         var tmp   = _filter;
-        var enter = ImGui.InputTextWithHint("##filter", "Filter...", ref tmp, 255, ImGuiInputTextFlags.EnterReturnsTrue);
+        var enter = ImGui.InputTextWithHint("##filter", "篩選...", ref tmp, 255, ImGuiInputTextFlags.EnterReturnsTrue);
         UpdateFilter(tmp);
 
         if (enter && _remainingItems.Count == 0)
diff --git a/Lifestream/OtterGui/Widgets/FilteredCombo.cs b/Lifestream/OtterGui/Widgets/FilteredCombo.cs
index f9f0de4..a397e32 100644
--- a/Lifestream/OtterGui/Widgets/FilteredCombo.cs
+++ b/Lifestream/OtterGui/Widgets/FilteredCombo.cs
@@ -147,7 +147,7 @@ public abstract class FilterComboBase<T>
 
         // Draw the text input.
         ImGui.SetNextItemWidth(width);
-        if (LowerString.InputWithHint("##filter", "Filter...", ref _filter))
+        if (LowerString.InputWithHint("##filter", "篩選...", ref _filter))
         {
             _filterDirty = true;
             if (SearchByParts)
