diff --git a/Lifestream/OtterGui/Log/Logger.cs b/Lifestream/OtterGui/Log/Logger.cs
new file mode 100644
index 0000000..1e05f99
--- /dev/null
+++ b/Lifestream/OtterGui/Log/Logger.cs
@@ -0,0 +1,70 @@
+using Serilog.Events;
+
+namespace OtterGui.Log;
+
+public sealed class Logger
+{
+    private readonly Serilog.ILogger _pluginLogger;
+    private readonly string _pluginName;
+    private readonly string _prefix;
+
+    public string PluginName => _pluginName;
+    public Serilog.ILogger MainLogger => _pluginLogger;
+
+    public Logger()
+    {
+        _pluginName = Assembly.GetCallingAssembly().GetName().Name ?? "Unknown";
+        _pluginLogger = Serilog.Log.ForContext("Dalamud.PluginName", _pluginName);
+        _prefix = $"[{_pluginName}] ";
+    }
+
+    public enum LogLevel
+    {
+        Excessive = LogEventLevel.Verbose,
+        Verbose = LogEventLevel.Verbose,
+        Debug = LogEventLevel.Debug,
+        Information = LogEventLevel.Information,
+        Warning = LogEventLevel.Warning,
+        Error = LogEventLevel.Error,
+        Fatal = LogEventLevel.Fatal,
+    }
+
+    public void Message(LogLevel level, string text)
+        => Serilog.Log.Write((LogEventLevel)level, "{Message}", _prefix + text);
+
+    public void Fatal(string text)
+        => _pluginLogger.Fatal("{Message}", _prefix + text);
+
+    public void Fatal(string format, params object?[] args)
+        => _pluginLogger.Fatal(_prefix + format, args);
+
+    public void Error(string text)
+        => _pluginLogger.Error("{Message}", _prefix + text);
+
+    public void Error(string format, params object?[] args)
+        => _pluginLogger.Error(_prefix + format, args);
+
+    public void Warning(string text)
+        => _pluginLogger.Warning("{Message}", _prefix + text);
+
+    public void Warning(string format, params object?[] args)
+        => _pluginLogger.Warning(_prefix + format, args);
+
+    public void Information(string text)
+        => _pluginLogger.Information("{Message}", _prefix + text);
+
+    public void Information(string format, params object?[] args)
+        => _pluginLogger.Information(_prefix + format, args);
+
+    public void Debug(string text)
+        => _pluginLogger.Debug("{Message}", _prefix + text);
+
+    public void Debug(string format, params object?[] args)
+        => _pluginLogger.Debug(_prefix + format, args);
+
+    public void Verbose(string text)
+        => _pluginLogger.Verbose("{Message}", _prefix + text);
+
+    public void Verbose(string format, params object?[] args)
+        => _pluginLogger.Verbose(_prefix + format, args);
+
+    [Conditional("EXCESSIVE_LOGGING")]
+    public void Excessive(string text)
+        => _pluginLogger.Verbose("{Message}", _prefix + text);
+
+    [Conditional("EXCESSIVE_LOGGING")]
+    public void Excessive(string format, params object?[] args)
+        => _pluginLogger.Verbose(_prefix + format, args);
+}
