diff --git a/Lifestream/OtterGui/Log/LazyString.cs b/Lifestream/OtterGui/Log/LazyString.cs
new file mode 100644
index 0000000..f468371
--- /dev/null
+++ b/Lifestream/OtterGui/Log/LazyString.cs
@@ -0,0 +1,16 @@
+namespace OtterGui.Log;
+
+public sealed class LazyString
+{
+    private readonly Func<string> _valueFactory;
+
+    public LazyString(Func<string> valueFactory)
+    {
+        _valueFactory = valueFactory ?? throw new ArgumentNullException(nameof(valueFactory));
+    }
+
+    public override string ToString()
+    {
+        return _valueFactory();
+    }
+}
