diff --git a/Lifestream/Lifestream/Tasks/Shortcuts/TaskPropertyShortcut.cs b/Lifestream/Lifestream/Tasks/Shortcuts/TaskPropertyShortcut.cs
index de2959f..41b9663 100644
--- a/Lifestream/Lifestream/Tasks/Shortcuts/TaskPropertyShortcut.cs
+++ b/Lifestream/Lifestream/Tasks/Shortcuts/TaskPropertyShortcut.cs
@@ -321,7 +321,7 @@ public static unsafe class TaskPropertyShortcut
     {
         enterApartment ??= C.EnterMyApartment;
         var a = GetApartmentAetheryteID();
-        var nextToMyApt = AgentHUD.Instance()->MapMarkers.Any(x => x.IconId.EqualsAny(60790u, 60792u) && Vector3.Distance(Player.Position, x.Position) < 50f) && Svc.Objects.Any(x => x.DataId == 2007402 && Vector3.Distance(x.Position, Player.Position) < 20f);
+        var nextToMyApt = AgentHUD.Instance()->MapMarkers.Any(x => x.IconId.EqualsAny(60790u, 60792u) && Vector3.Distance(Player.Position, new Vector3(x.X, x.Y, x.Z)) < 50f) && Svc.Objects.Any(x => x.DataId == 2007402 && Vector3.Distance(x.Position, Player.Position) < 20f);
         P.TaskManager.BeginStack();
         try
         {
