diff --git a/Lifestream/Lifestream/Lang.cs b/Lifestream/Lifestream/Lang.cs
index ea70ec6..07e6997 100644
--- a/Lifestream/Lifestream/Lang.cs
+++ b/Lifestream/Lifestream/Lang.cs
@@ -62,7 +62,7 @@ internal static class Lang
         Regex.Replace(Svc.Data.GetExcelSheet<EObjName>().GetRow(2004353).Singular.GetText(), @"\[.*?\]", "")
     ];
 
-    internal static readonly string[] EnterWorkshop = ["Move to the company workshop", "地下工房に移動する", "移动到部队工房", "移動到部隊工房", "Die Gesellschaftswerkstätte betreten", "Aller dans l'atelier de compagnie", "지하공방으로 이동"];
+    internal static readonly string[] EnterWorkshop = ["Move to the company workshop", "地下工房に移動する", "移动到部队工房", "移動到公會工坊", "Die Gesellschaftswerkstätte betreten", "Aller dans l'atelier de compagnie", "지하공방으로 이동"];
 
     internal static Dictionary<WorldChangeAetheryte, string> WorldChangeAetherytes = new()
     {
